From dba9ba544a451eafdae2c798faf54b2b20450911 Mon Sep 17 00:00:00 2001
From: Van Do <van.do.xw@rvc.renesas.com>
Date: Fri, 9 Feb 2018 10:57:50 +0700
Subject: [PATCH 182/909] dt-bindings: pwm: rcar: Add bindings for R-Car E3
 support

commit b895c83e488099f19fedf4d7056aef82fadaf134 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Document support for PWM in the Renesas R-Car E3.

Signed-off-by: Van Do <van.do.xw@rvc.renesas.com>
Signed-off-by: Kazuya Mizuguchi <kazuya.mizuguchi.ks@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt b/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt
index 35a3b9761ee5..0dc26ca64339 100644
--- a/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt
+++ b/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.txt
@@ -13,6 +13,7 @@ Required Properties:
  - "renesas,pwm-r8a7796": for R-Car M3-W
  - "renesas,pwm-r8a77965": for R-Car M3-N
  - "renesas,pwm-r8a77995": for R-Car D3
+ - "renesas,pwm-r8a77990": for R-Car E3
 - reg: base address and length of the registers block for the PWM.
 - #pwm-cells: should be 2. See pwm.txt in this directory for a description of
   the cells format.
-- 
2.17.1

